docs/reference/commandline/run.md

```diff
@@ -302,21 +296,26 @@ For additional information on working with labels, see [*Labels - custom
 metadata in Docker*](../../userguide/labels-custom-metadata.md) in the Docker User
 Guide.
 
-### Add link to another container (--link)
+### Connect a container to a network (--net)
+
+When you start a container use the `--net` flag to connect it to a network.
+This adds the `busybox` container to the `mynet` network.
 
-    $ docker run --link /redis:redis --name console ubuntu bash
+```bash
+$ docker run -itd --net=my-multihost-network busybox
+```
 
-The `--link` flag will link the container named `/redis` into the newly
-created container with the alias `redis`. The new container can access the
-network and environment of the `redis` container via environment variables.
-The `--link` flag will also just accept the form `<name or id>` in which case
-the alias will match the name. For instance, you could have written the previous
-example as:
+If you want to add a container to a network after the container is already
+running use the `docker network connect` subcommand.
 
-    $ docker run --link redis --name console ubuntu bash
+You can connect multiple containers to the same network. Once connected, the
+containers can communicate easily needinly only another container's IP address
```

Comment:
"easily needinly only"? probably should've been "needing"?

**NOTE** service discovery (i.e. referencing a container by *name* on a network) will *not* be supported on the default "bridge" network, and will *only* work for containers that have been explicitly given a name (through `--name`)